### PR TITLE
Initial v0.1 version of the app.

### PR DESCRIPTION
**Pull request by kstepanovdev**

### CODE REVIEW

**Review by TheBestTvarynka — APPROVED** [review]

File: src/main.rs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+pub fn initialize_panic_handler() {
+    std::panic::set_hook(Box::new(|panic_info| {
+        better_panic::Settings::auto()
+            .most_recent_first(false)
+            .lineno_suffix(true)
+            .verbosity(better_panic::Verbosity::Full)
+            .create_panic_handler()(panic_info);
+        std::process::exit(libc::EXIT_FAILURE);
```
Comment (silvestrpredko) [inline src/main.rs:48]: Will it work with any system? I mean for example Windows?

File: src/event.rs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    handler: thread::JoinHandle<()>,
+}
+
+impl EventHandler {
+    pub fn new(tick_rate: u64) -> Self {
+        let tick_rate = Duration::from_millis(tick_rate);
+        let (sender, receiver) = mpsc::channel();
+        let handler = {
+            let sender = sender.clone();
+            thread::spawn(move || {
```
Comment (silvestrpredko) [inline src/event.rs:31]: Would be nice to save the handler for this thread.

**Review by silvestrpredko — APPROVED** [review]
Overall looks awesome.

P.S.
better_panic - nice crate, I haven't heard about it before.

**Review by 0x07C0 — APPROVED** [review]

**Review by Jujumba — APPROVED** [review]
Looks cool!

**Review by SupperZum — APPROVED** [review]
I like it very much!

**Review by danik-tro — APPROVED** [review]
Looks great!

**Pull request merged**